### PR TITLE
Implement standalone mobile apps building with Expo

### PR DESCRIPTION
**Pull request by vlasenko**
Expo free build service is used to create standalone signed mobile applications without need for compiling native code on developer machine. Production JavaScript bundles generated via Webpack are bundled with Expo-compiled native code together as final mobile applications

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,6 +297,12 @@ While developing, you will probably rely mostly on `yarn watch`; however, there
   yarn start
   ```
 
+### Building standalone mobile apps for Play Store and App Store
+1. Compile project for production with `ios` and `android` set to `true` in `app.json`.
+2. Run `yarn exp ba` to launch building signed `.apk` or `yarn exp bi` for signed `.iap`.
```
Comment (mairh) [inline README.md]: We need to run `yarn watch` first before executing these 2 commands, right?

Comment (vlasenko) [inline README.md]: @mairh No. For production builds you need to run `yarn build` and then `yarn exp ba` for Android or `yarn exp bi` for iOS. `yarn exp ba/bi` will run Expo server that serves production JavaScript bundles.

Comment (mairh) [inline README.md]: Maybe it's more clear `Compile project `yarn build` for production with `ios` and `android` set to `true` in `app.json`.`

Comment (vlasenko) [inline README.md]: @mairh Yes, feel free to fix the docs, they are not very clear at this right now, thanks Ujjwal!

**Pull request merged**